api/tests/unit/sales_dashboard/test_unit_sales_dashboard_views.py

```diff
@@ -75,18 +75,16 @@ def test_get_organisation_info__get_event_list_for_organisation(
 
 def test_list_organisations_search_by_name(
     organisation: Organisation,
-    client: Client,
-    admin_user: FFAdminUser,
+    superuser_client: Client,
```

Comment:
What is this `superuser_client` fixture and why can't we use `admin_client` instead?

Reply:
Superuser_client is `django.test.client.Client` and admin_client is `rest_framework.test.APIClient` i,e: admin_client will not work with `@staff_member_required`